domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/statementofmeans/Residence.java

```diff
@@ -41,6 +43,10 @@ public ResidenceType getType() {
         return type;
     }
 
+    public String getTypeDescription() {
```

Comment:
The same here. Please make sure all of that is removed from remaining places as well.